tslint.json

```diff
@@ -61,7 +61,7 @@
       "single",
       "avoid-escape"
     ],
-    "radix": true,
+    "radix": false,
```

Comment:
The default is base 10 so I'm not sure why the linter thinks it's necessary to have to put the radix value there so I just shut it off.